src/i18n/locale-en.json

```diff
@@ -925,6 +926,7 @@
     "import.last.import.settings": "Last import settings",
     "import.mode.not.supported.constraint": "This mode is not supported when importing multiple items.",
     "import.enable.replace.option": "Enable this to replace the data in one or more graphs with the imported data.",
+    "import.context.link": "Context link",
```

Comment:
You should also update the file `test-cypress/fixtures/locale-en.json`

Reply:
Updated.